keyboards/crkbd/rev1/common/keymaps/via/keymap.c

```diff
@@ -27,7 +27,7 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
   //|--------+--------+--------+--------+--------+--------|                    |--------+--------+--------+--------+--------+--------|
       KC_LSFT,    KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,                         KC_N,    KC_M, KC_COMM,  KC_DOT, KC_SLSH,  KC_ESC,
   //|--------+--------+--------+--------+--------+--------+--------|  |--------+--------+--------+--------+--------+--------+--------|
-                                          KC_LGUI, FN_MO13,  KC_SPC,     KC_ENT, FN_MO23, KC_RALT
+                                          KC_LGUI,   MO(1),  KC_SPC,     KC_ENT,   MO(2), KC_RALT
```

Comment:
Would prefer that this remains as it was.  the `FN_MOxx` keycodes work as a tri layer keycode.  This works better than using the MO keycodes. 

https://github.com/qmk/qmk_firmware/blob/b0024470ed347271e29b2de4d0442f855a26345c/quantum/via.c#L163-L183

```suggestion
                                          KC_LGUI, FN_MO13,  KC_SPC,     KC_ENT, FN_MO23, KC_RALT
```

Reply:
I see. Hou to set `FN_MOxx` on VIA Configurator?

Reply:
It's the Fn1 and Fn2 buttons in "Layers" (the first 2 keys, actually).

Reply:
Thanks! I see.